app/src/panel/models/page/blueprint/pages.php

```diff
@@ -22,7 +22,7 @@ class Pages extends Obj {
   public function __construct($params = array()) {
 
     if($params === true) {
-      $this->template = blueprint::all();
+      $this->template = true;
```

Comment:
Oh, do we? I didn't know we already used `true` with the meaning `all`. If so, that's completely fine. :)